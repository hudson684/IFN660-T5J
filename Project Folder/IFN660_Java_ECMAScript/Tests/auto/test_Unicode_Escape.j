'\"'
\u0022
\u0054
"End of test"
\u005cu005a