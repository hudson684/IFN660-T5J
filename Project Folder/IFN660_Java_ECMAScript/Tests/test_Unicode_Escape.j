//This is just for messing around with your testing.

true 

Start
\"
\ 
\u0022
\u0054

u00ef

"Middle"

9
99999

123310029
"End of test"