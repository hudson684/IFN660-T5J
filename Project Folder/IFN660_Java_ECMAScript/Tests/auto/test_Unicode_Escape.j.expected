CharacterLiteral ('\"')
UnicodeInputCharacter (\u0022)
UnicodeInputCharacter (\u0054)
StringLiteral ("End of test")
UnicodeInputCharacter (\u005c)
IDENT (u005a)
EOF